mobify/magnifik
# Adding touch scroll

Status: **Ready for Review**
Owner: @yourpalsonja
Reviewers: @jansepar @donnielrt @marlowpayne 

## Changes
- Added `-webkit-overflow-scrolling: touch` for a jank-free scrolling experience!

## Todos:

## How to Test
- Open the test file and click on the image. (Got it hosted at [new-silk.surge.sh](new-silk.surge.sh) if you wanna check on your phone)
